src/Nancy/RequestHeaders.cs

```diff
@@ -372,17 +428,19 @@ private static IEnumerable<INancyCookie> GetNancyCookies(IEnumerable<string> coo
 
         private IEnumerable<string> GetValue(string name)
         {
-            return this.GetValue(name, x => x);
+            return this.GetValue(name, x => x, new string[] {});

```

Comment:
`Array.Empty<string>`? Or is that a 4.6+ API? Maybe we should have an `EmptyArrayCache<T>` class, similar to [MVC's TaskCache](https://github.com/aspnet/Mvc/blob/4f709bdbfdad5150ca941808ecbe3a15b9cc9089/src/Microsoft.AspNetCore.Mvc.Core/Internal/TaskCache.cs)? Either that, or we simply cache an empty string array in this class.